pages/workflow.md

```diff
@@ -1,18 +1,31 @@
 
-Take plenty of screenshots to document your process...
-    * **Mac** press 'command' + 'shift' + '4' and then drag a rectangle to take a screenshot of whatever you select... On
-    * **Windows**, click the 'Start' button then type 'Snipping' and select Snipping Tool, then click 'New' and drag a rectangle to take a screenshot of whatever you select.
+The first visualization being the main visualization on immigration from 19th century until 21st century which had some laws that effected immigration throughout the timeline. For example, there was a drop of immigration during the 1917 immigration act, which then the drop kept decreasing from 1917 until events leading to WWII. There was also the 1953 Refugee Relief Act which showed from the graph that little by little throughout decades’ immigrations started to Increase tremendously. The Visualization showed the major shifts of migration to the United States starting from the year 1825, which showed the cause being United States Laws. The second visualization is about immigration Pre-World War II by European nations. The Visualization showed that migration was decreasing by a huge margin leading to World War II which makes sense since Wars against other nations would cause some National security concerns. It first started dropping from the events of World War I but then closer to the second World War there was stricter immigration policies. The last visualization was on wars and events that effected immigration. There was a big growth during 1880 from Asians because of the Angell Treaty of 1880 which helped the growth of Asian migration during the late 1800’s. 
+
+The Visualization showed that some sort of growth of Immigration during the American Civil War which would mean that the United States would use some of the Immigrations to fight for citizenships and help them start a new life. This Visualizations showed the increase and decrease of immigrants throughout the years from major events and wars that would cause such drastic chances of immigration.
+
+---
+
+# Research Work Flow
+
+As for my work flow, I was tasked on the historical and digital scholarship of the project. We as a group found the data for our project which left me with just the work on researching laws and reasons for immigration. I was having issues finding arguments about immigration law as some articles were too broad or just not fitting the narrative of the project. JSTOR's advanced search functions proved the most useful for looking up topics on the affects of laws on immigration. There was multiple journal articles with arguements for and against important laws. Other credible sources as annotated in the biliography, showcase affects as well on immigration. 
```

Comment:
Again, change this to the royal we, and here, don't talk about your individual assignment. Talk about what 'the group' did to research the topic.

Also, WHY NOT MENTION GITHUB in your workflow? After all, everyone in the group has to make their changes on github using Markdown, none of which you mention anywhere. That's a significant part of the skills you had to learn!